python_modules/dagster/dagster_tests/core_tests/selector_tests/test_subset_selector.py

```diff
@@ -272,6 +265,14 @@ def asset_2(my_asset):
     return my_asset
 
 
-asset_selection_job = AssetGroup([my_asset, asset_2]).build_job(
-    "asset_selection_job", executor_def=execute_in_process_executor
-)
+@repository
+def asset_house():
+    return [
+        my_asset,
+        asset_2,
+        define_asset_job("asset_selection_job", selection="*", executor_def=in_process_executor),
+    ]
+
+
+def get_asset_selection_job():
+    return asset_house.get_job("asset_selection_job")
```

Comment:
Instead of constructing the repository, you could just do `return build_assets_job([my_asset, asset_2])`.